orion/model.py

```diff
@@ -95,8 +95,12 @@ def validate(self, value):
 
 class Dataset(Document, MongoUtils):
     name = fields.StringField(required=True)
-    signal_set = fields.StringField(required=True)
-    satellite_id = fields.StringField()
+    entity_id = fields.StringField()
```

Comment:
Why is this called `entity_id` instead of `satellite_id`?

Reply:
I think `entity_id` is a more general term, as we could also have other datasets that do not come from satellites in the future.